### PR TITLE
[WIP] Translation Vietnam language in menu item

### PR DESCRIPTION
**Pull request by motokito**
Complete translation the menu in Vietnam language 🎅. Please take a look to merge it. THX :cow:

Now look it like these screenshoots:
2016-09-26_18h28_10
2016-09-26_18h28_39
2016-09-26_18h29_05
2016-09-26_18h29_21
2016-09-26_18h29_38
2016-09-26_18h29_58
2016-09-26_18h30_44
2016-09-26_18h31_49
2016-09-26_18h32_35
2016-09-26_18h33_11

- [X] Change in CHANGELOG.md described
- [X] Screenshots added (for bigger UI changes)
- [X] Manually tested changed features in running JabRef
- [ ] Internal SQ

### CODE REVIEW

**Comment by mlep** [comment]
Could/should @motokito  be added to the team @JabRef/translators?

**Comment by Siedlerchr** [comment]
@mlep  If he's interested in doing further translations tasks in future, we can add him.

**Comment by mlep** [comment]
@Siedlerchr: I think the thumb up added to your comment by @motokito tells all.
Welcome on board @motokito !

**Comment by lenhard** [comment]
A complete translation for Vietnamese would be cool. @motokito Can you translate all items in the language file for Vietnamese in this PR?

**Comment by motokito** [comment]
First i will translate the menu item. Then I will look ewaht to do next 😄

**Comment by lenhard** [comment]
Ok, but to avoid that we have to review 500 PRs with 2 new lines (My guess from looking at the file is that 1000 entries are missing in vietnamese, but this might be very inaccurate), I'd suggest the following: We leave this PR are open until you either have translated all items or you say yourself that you are ultimately done with the translation. 

It is really absolutely fine if you do not translate everything, it would just be annoying if we bombard JabRef with small PRs for the same task. Reviewing PRs consumes time, even if they are very simple. In the case of localizations, one big batch is very much preferable over many small contributions.

**Comment by mlep** [comment]
Since this PR is only about the menu items, it is about 30 lines to be translated (see https://github.com/JabRef/jabref/pull/2010).

**Comment by motokito** [comment]
All conflict are fixed. :hibiscus:
Please take a look to merge . THX :koala:

File: src/main/resources/l10n/JabRef_fr.properties [hunk 1]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Montrer_les_colonnes_supplémentaires
 Parsing_error=Erreur_de_traitement
 illegal_backslash_expression=Expression_avec_barre_oblique_inversée_(\\)_illégale
 
-Move_to_group=Déplacer_vers_le_groupe
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_fr.properties]: Translation removed? Why?

File: src/main/resources/l10n/JabRef_it.properties [hunk 2]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Mostra_le_colonne_supplementari
 Parsing_error=Errore_di_elaborazione
 illegal_backslash_expression=Espressione_con_barra_retroversa_illegale
 
-Move_to_group=Sposta_nel_gruppo
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_it.properties]: Same remark

File: src/main/resources/l10n/JabRef_de.properties [hunk 3]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Extraspalten_anzeigen
 Parsing_error=Syntaxfehler
 illegal_backslash_expression=ungültiger_Backslash-Ausdruck
 
-Move_to_group=Verschieben_in_Gruppe
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_de.properties]: Translation removed? Why?

File: src/main/resources/l10n/JabRef_in.properties [hunk 4]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Tampilkan_kolom_ekstra
 Parsing_error=Kesalahan_penguraian
 illegal_backslash_expression=Ekspresi_tanda_garis_miring_terbalik_tidak_sah
 
-Move_to_group=Pindah_ke_grup
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_in.properties]: Same remark

File: src/main/resources/l10n/JabRef_pt_BR.properties [hunk 5]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Mostrar_colunas_extra
 Parsing_error=Erro_de_interpretação
 illegal_backslash_expression=Expressã_ilegal_com_contrabarra
 
-Move_to_group=Mover_para_grupo
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_pt_BR.properties]: Same remark

File: src/main/resources/l10n/JabRef_ru.properties [hunk 6]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Показывать_столбцы_доп.сведений
 Parsing_error=Ошибка_анализа
 illegal_backslash_expression=недопустимое_выражение_с_обратной_косой_чертой
 
-Move_to_group=Переместить_в_группу
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_ru.properties]: Same remark

File: src/main/resources/l10n/JabRef_sv.properties [hunk 7]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Visa_extrakolumner
 Parsing_error=
 illegal_backslash_expression=
 
-Move_to_group=Flytta_till_grupp
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_sv.properties]: Same remark

File: src/main/resources/l10n/JabRef_tr.properties [hunk 8]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Ekstra_sütunları_göster
 Parsing_error=Çözümleme_hatası
 illegal_backslash_expression=kuraldışı_ters_bölü_ifadesi
 
-Move_to_group=gruba_taşı
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_tr.properties]: Same remark

File: src/main/resources/l10n/JabRef_zh.properties [hunk 9]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=显示额外的列
 Parsing_error=解析错误
 illegal_backslash_expression=
 
-Move_to_group=移动到分组
+Move_to_group=
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_zh.properties]: Same remark

File: src/main/resources/l10n/JabRef_vi.properties [hunk 10]
```diff
@@ -1482,7 +1482,7 @@ Error_opening_autosave_of_'%0'._Trying_to_load_'%0'_instead.=Lỗi_khi_mở_tậ
 Formatter_not_found\:_%0=Không_tìm_thấy_trình_định_dạng\:_%0
 Clear_inputarea=Xóa_trong_vùng_nhập_liệu
 
-Automatically_set_file_links_for_this_entry=Tự_động_thiết_lập_các_liên_kết_tập_tin_cho_mục_này
+Automatically_set_file_links_for_this_entry=
```
Comment (lenhard) [inline src/main/resources/l10n/JabRef_vi.properties]: Same here. Why is this entry removed?

File: src/main/resources/l10n/JabRef_fr.properties (hunk 1 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_fr.properties]: Sorry. I think this problem is happened, during i merge and assign gradle update 😄. But i have redone it 👍

File: src/main/resources/l10n/JabRef_it.properties (hunk 2 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_it.properties]: Done 😄

File: src/main/resources/l10n/JabRef_de.properties (hunk 3 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_de.properties]: Done 😄

File: src/main/resources/l10n/JabRef_in.properties (hunk 4 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_in.properties]: Done 😄

File: src/main/resources/l10n/JabRef_ja.properties [hunk 11]
```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=追加列を表示
 Parsing_error=解析エラー
 illegal_backslash_expression=バックスラッシュ表現が不正です
 
-Move_to_group=グループに移動
+Move_to_group=
```
Comment (motokito) [inline src/main/resources/l10n/JabRef_ja.properties]: Done 😄

File: src/main/resources/l10n/JabRef_pt_BR.properties (hunk 5 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_pt_BR.properties]: Done 😄

File: src/main/resources/l10n/JabRef_ru.properties (hunk 6 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_ru.properties]: Done 😄

File: src/main/resources/l10n/JabRef_sv.properties (hunk 7 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_sv.properties]: Done 😄

File: src/main/resources/l10n/JabRef_tr.properties (hunk 8 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_tr.properties]: Done 😄

File: src/main/resources/l10n/JabRef_zh.properties (hunk 9 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_zh.properties]: Done 😄

File: src/main/resources/l10n/JabRef_vi.properties (hunk 10 above)
Comment (motokito) [inline src/main/resources/l10n/JabRef_vi.properties]: This problem is happened during i merge and gradle update 😄 I have done it 👍

**Comment by Siedlerchr** [comment]
Okay, loogs good now  I will merge this version in. 
Best you create then a new branch for future updates

**Comment by motokito** [comment]
THX 😄 
I will do it 🐨

**Pull request merged**